conf/modules/distributed_circular_formation.xml

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="distributed_circular_formation" dir="dcf">
+  <doc>
+    <description>Distributed algorithm for circular formations.
+    </description>
+    <section name="Parameters" prefix="DCF_">
+      <define name="MAX_NEIGHBORS" value="4" description="Maximum number of accepted neighbors for an aircraft"/>
+    </section>
+  </doc>
+
+  <settings name="DCF">
+    <dl_settings>
+      <dl_settings NAME="DCF">
+        <dl_settings NAME="Control">
+          <dl_setting MAX="20" MIN="0" STEP="0.2" VAR="dcf_control.k" shortname = "Gain" param="DCF_GAIN_K"/>
+          <dl_setting MAX="200" MIN="0" STEP="1" VAR="dcf_control.radius" shortname = "Radius" param="DCF_RADIUS"/>
+          <dl_setting MAX="5000" MIN="0" STEP="1" VAR="dcf_control.timeout" shortname = "Timeout" param="DCF_TIMEOUT"/>
+        </dl_settings>
```

Comment:
These variables should also be described above in the description as they can also be set in airframe files.

Reply:
They are described in detail the wiki:

https://wiki.paparazziuav.org/wiki/Module/guidance_vector_field#Fully_distributed_circular_formations

A reference in the head of the file will be enough?